go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/livekit/livekit-server
 
-go 1.20
+go 1.21
```

Comment:
Do we know what change (a module update I suppose?) triggered this requirement for go 1.21? I'm not entirely Sure whether bumping this up is something we care about or not

Reply:
I believe it might be my new Go version + `go mod tidy` triggering this. It updated to 1.22 recently.

It also added the `toolchain` directive, but I intentionally removed it since it's known to cause issues with some code analysis tooling which doesn't support recent Go versions.

I believe it should be safe. Never seen it causing issues :man_shrugging: